### PR TITLE
doc: removed catalogs_flexible_pricing related docs

### PR DESCRIPTION
**Pull request by shihanxiong**
# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [x] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [ ] Check that you haven't removed any images (replacing an image with an updated one of the same name is fine), as this breaks the French site
- [ ] Check that all links work.
- [ ] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [ ] Tag @josh-mccrowell-braze and @bre-fitzgerald as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the internal reviewing chart to tag the appropriate reviewer.
- [ ] If the documentation involves a 1) paid SKU, 2) a third party, 3) SMS, 4) AI, or 5) privacy, ensure that Maria Maldonado on the Legal team has signed off.
- [ ] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub)
- [Image Style Guide](https://docs.google.com/document/d/e/2PACX-1vRJSkwcjmjrTfLDagZccLpOMMyh5NN5SXRZSjz12cRAHbX4OrUmhvCmYpf_p5YB-9r4_jSOQLkicQIH/pub)
- [Styling Test Page](https://www.braze.com/docs/home/styling_test_page/)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Vercel environment by clicking the preview link in the vercel-bot comment in your PR.
</details>

<details>
<summary>❗ ATTN: Internal Reviewing Chart </summary>
<br>
<b>Work at Braze and not sure who to tag for review?</b> <br>Before tagging @josh-mccrowell-braze or @bre-fitzgerald for a general review, reference the following chart to see if a specific product vertical/reviewer applies to your pull request.
<br><br>
<table>
<tr>
    <td><b>Reviewer</b></td>
    <td><b>Product Vertical</b></td>
  </tr>
  <tr>
    <td>@josh-mccrowell-braze</td>
    <td>Monolith Deployments<br>Quality Infrastructure<br>Platform Infrastructure<br>Datalake<br>SDKs<br>Currents</td>
  </tr>
  <tr>
    <td>@bre-fitzgerald</td>
    <td>Intelligence<br>In-App Messages<br>Channels<br>FIX</td>
  </tr>
  <tr>
    <td>@lydia-xie</td>
    <td>Ingestion<br>Core Objects<br>Core Messaging<br>Messaging Experience<br>Message Components<br>Email (Composition and Infrastructure) (tag @rachel-feinberg for Liquid use cases)</td>
  </tr>
  <tr>
    <td>@rachel-feinberg</td>
    <td>Customer Lifecycle, Identity and Permissions<br>SMS<br>User Targeting<br>Reporting</td>
  </tr>
</table>
</details>

### CODE REVIEW

**Review by HVisagie — APPROVED** [review]

File: _lang/fr/_api/endpoints/catalogs/catalog_management/synchronous/post_create_catalog.md [hunk 1]
```diff
@@ -185,7 +185,6 @@ Le tableau suivant répertorie les erreurs renvoyées possibles et les étapes d
 | `invalid-field-types` | Assurez-vous que les types des champs sont valides. |
 | `field-names-too-large` | La limite de caractères d’un nom de champ est de 250. |
 | `field-names-not-unique` | Le même nom de champ est référencé deux fois. |
```
Comment (lydia-xie) [inline _lang/fr/_api/endpoints/catalogs/catalog_management/synchronous/post_create_catalog.md:187]:
```suggestion
| `field-names-not-unique` | Le même nom de champ est référencé deux fois. |
| `reached-company-catalogs-limit` | Le nombre maximum de catalogues est atteint. Contactez votre gestionnaire de compte Braze pour plus d’informations. |
```

File: _lang/fr/_api/endpoints/catalogs/catalog_items/synchronous/post_create_catalog_item.md [hunk 2]
```diff
@@ -114,8 +114,6 @@ Le tableau suivant répertorie les erreurs renvoyées possibles et les étapes d
 | `item-already-exists` | Ce produit existe déjà dans le catalogue. |
 | `invalid-fields` | Confirmez que les champs de la requête existent dans le catalogue. |
 | `filtered-set-field-too-long` | La valeur du champ est utilisée dans un ensemble filtré qui dépasse la limite de caractères pour un produit. |
```
Comment (lydia-xie) [inline _lang/fr/_api/endpoints/catalogs/catalog_items/synchronous/post_create_catalog_item.md:116]:
```suggestion
| `filtered-set-field-too-long` | La valeur du champ est utilisée dans un ensemble filtré qui dépasse la limite de caractères pour un produit. |
| `already-reached-catalog-item-limit` | Le nombre maximum de catalogues est atteint. Contactez votre gestionnaire de compte Braze pour plus d’informations. |
| `already-reached-company-item-limit` | Le nombre maximum de produits du catalogue est atteint. Contactez votre gestionnaire de compte Braze pour plus d’informations. | 
```

File: _lang/fr/_api/endpoints/catalogs/catalog_management/synchronous/post_create_catalog.md [hunk 3]
```diff
@@ -185,6 +185,7 @@ Le tableau suivant répertorie les erreurs renvoyées possibles et les étapes d
...
+| `reached-company-catalogs-limit` | Le nombre maximum de catalogues est atteint. Contactez votre gestionnaire de compte Braze pour plus d’informations. |
```
Comment (shihanxiong) [inline _lang/fr/_api/endpoints/catalogs/catalog_management/synchronous/post_create_catalog.md:188]: @lydia-xie I think we should remove this line

Comment (lydia-xie) [reply _lang/fr/_api/endpoints/catalogs/catalog_management/synchronous/post_create_catalog.md:188]: Hey @shihanxiong - Our French docs get translated from our English docs, so this edit will be included!

Comment (shihanxiong) [reply _lang/fr/_api/endpoints/catalogs/catalog_management/synchronous/post_create_catalog.md:188]: Ah gotcha! Thanks

**Pull request merged**